resources/public/api/conf/1.0/retrieve_eops_obligations.raml

```diff
@@ -0,0 +1,41 @@
+is:
+  - headers.acceptJson
+  - headers.contentTypeJson
+  - headers.testScenarioHeader
+  - errors.formatNino
+  - errors.FormatTypeOfBusiness
+  - errors.FormatBusinessId
+  - errors.FormatFromDate
+  - errors.FormatToDate
+  - errors.FormatStatus
+  - errors.MissingFromDate
+  - errors.MissingToDate
+  - errors.RangeToDateBeforeFromDate
+  - errors.MissingTypeOfBusiness
+  - errors.RuleDateRangeInvalid
+  - errors.RuleFromDateNonSupported
+  - errors.clientOrAgentNotAuthorised
+  - errors.notFound
+  - queryParameters.typeOfBusiness
+  - queryParameters.incomeSourceId
```

Comment:
I noticed this for all the RAML, the queryParameters.incomeSourceId Raml Needs to have the Regex fixed to be ^X[a-zA-Z0-9]{1}IS[0-9]{12}$